src/_includes/config/setup-cron_2.3_how-to.md

```diff
@@ -16,7 +16,7 @@ To create the Magento crontab:
 
 Use `--force` to rewrite an existing Magento crontab.
 
- {:.bs-callout-info}
+{:.bs-callout-info}
```

Comment:
The use of this style of note causes the info icon to obscure the bullet point. Not a show-stopper, but looks odd.

Reply:
Noted